lib/devise.rb

```diff
@@ -145,6 +145,10 @@ module Test
   mattr_accessor :reconfirmable
   @@reconfirmable = true
 
+  #TODO- we should turn it true on a future major release
+  mattr_accessor :new_email_confirmable
```

Comment:
What do you think of using `send_reconfirmation_instructions` for this accessor instead?